app/package-lock.json

```diff
@@ -4948,6 +4828,17 @@
         "node": ">=0.10.0"
       }
     },
+    "node_modules/react-blockies": {
```

Comment:
The package-lock.json shows `react-blockies` as a dependency, but it's not listed in package.json. This should be cleaned up, as it also generates conflicts with main branch.